releasenotes/notes/add-control-c-handler-fb2967d1ba8ee495.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    Add the possibility to terminate guake with ctrl+c
```

Comment:
can you add "from the starting console, so it won't affect ctrl+c from inside guake's terminals" or something like that?

Reply:
I'm sorry I have been on a little trip the past few days. Thanks for making the change  👍 

> Add the possibility to terminate guake with `Ctrl+c` on terminal where Guake has been launched

Sounds good!

Reply:
no pbl